Invoke-LiveResponse.psm1

```diff
@@ -600,6 +605,48 @@
         $ForensicCopyText = $ForensicCopyText + "`t`tUser Artefacts`n"
     }
 
+    # Recycle bin collection

```

Comment:
I'll hold off on merging this until I add my bulk changes Im about to add in as I think it would make this code significantly smaller.

What do you think of using drive enumeration of:
[System.IO.DriveInfo]::getdrives() | where-object {$.DriveType -eq "Fixed" -and $.DriveFormat -eq "NTFS"} | Select-Object -property Name
instead? This give more filter options and compatible accross win7 +

Reply:
Sure, that seems to work well across multiple OSes.